src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/Email.java

```diff
@@ -56,4 +61,8 @@ public int hashCode() {
     public EmailMetadata getEmailMetadata() {
         return emailMetadata;
     }
+
+    public List<Address> getRecipientCC() {
+        return this.recipientCC;
+    }
```

Comment:
```suggestion
    public List<Address> getRecipientCC() {
        return this.emailMetadata.recipients().cc();
    }
```
We have no need for a new instance field here.